app/helpers/i18n_helpers.rb

```diff
@@ -26,6 +26,23 @@ def t_attr(attribute, model = nil)
     model_class.human_attribute_name(attribute)
   end
 
+  # Returns translated name for the given +attribute+ in pluralized form.
+  #
+  # This is the pluralization version of t_attr.
+  #
+  # There needs to be one and other keys in:
+  # de.activerecord.attributes.patient.bill.one = Rechnung
+  # de.activerecord.attributes.patient.bill.other = Rechnungen
+  #
+  # Example:
+  #   t_attr('bill', Patient) => 'Rechnung'
+  #   t_attr('bill')          => 'Rechnungen' # when called in patients_controller views
+  #
+  def t_attrs(attribute, model = nil)
+    return t_attr(attribute, model).pluralize if locale.to_s == 'en'
+    I18n.translate(attribute, :scope => [:activerecord, :attributes, get_model_key(model)], count: 2)
```

Comment:
But i18n only resolves ':zero', ':one' and ':other'. 

As the railsguide sais, other can insert count as '%{count}' in the locale. But as far as I've played around with that thing, it didn't seem to work with ':other' inside 'scope: [:activerecord, ....]'

But anyway: if I didn't get the i18n way completely wrong, i don't see how I could fix this problem for those languages that have many kind of plurals. As far as I can see, i18n as a whole wouldn't support that.